app/views/course_user_data/sudo.html.erb

```diff
@@ -1,10 +1,51 @@
+<% content_for :javascripts do %>
+
+  <script type="application/javascript">
```

Comment:
Replace this chunk with `<%= render partial: "common/autocomplete", locals: { hiddenCUDField: "#sudo_id" } %>` once #1532 merged